infrastructure/service-bus.tf

```diff
@@ -1,35 +1,49 @@
 #HMC to Hearings API
-module "servicebus-subscription" {
-  source                = "git@github.com:hmcts/terraform-module-servicebus-subscription?ref=master"
-  name                  = "hmc-to-sscs-subscription-${var.env}"
-  namespace_name        = "hmc-servicebus-${var.env}"
-  topic_name            = "hmc-to-cft-${var.env}"
-  resource_group_name   = "hmc-shared-${var.env}"
+locals {
+  namespace_name      = "hmc-servicebus-${var.env}"
+  resource_group_name = "hmc-shared-${var.env}"
+  topic_name          = "hmc-to-cft-${var.env}"
+  key_vault_secrets = [
+    "hmc-servicebus-connection-string",
+    "hmc-servicebus-shared-access-key"
+  ]
 }
 
-data "azurerm_key_vault" "hmc-key-vault" {
-  name                = "hmc-${var.env}"
-  resource_group_name = "hmc-shared-${var.env}"
+module "servicebus_subscription" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-subscription?ref=master"
+  name                = "hmc-to-sscs-subscription-${var.env}"
+  namespace_name      = local.namespace_name
+  topic_name          = local.topic_name
+  resource_group_name = local.resource_group_name
 }
 
-data "azurerm_key_vault_secret" "hmc-servicebus-connection-string" {
-  key_vault_id = data.azurerm_key_vault.hmc-key-vault.id
-  name         = "hmc-servicebus-connection-string"
+resource "azurerm_servicebus_subscription_rule" "rule_sscs" {
```

Comment:
(Minor) The rule name could be a bit more descriptive

Reply:
I am unsure what the rule is for so added a simple name. @amoslv1 can update it if you would like?

Reply:
changed to topic_filter_rule_sscs